controls/2_1_inetd_services.rb

```diff
@@ -25,6 +25,10 @@
   tag cis: 'distribution-independent-linux:2.1.1'
   tag level: 1
 
+  only_if('inetd/xinetd config exists') {
+    file('/etc/xinetd.conf').exist? || file('/etc/inetd.conf').exist?
+  }
+
```

Comment:
```suggestion
  only_if('inetd/xinetd config exists') do
    file('/etc/xinetd.conf').exist? || file('/etc/inetd.conf').exist?
  end

```

I was getting rubocop warnings when using '{...}' syntax on multiple lines.